peachjam/static/stylesheets/components/_navbar.scss

```diff
@@ -2,7 +2,6 @@
   .nav-item,
   .nav-link {
     color: black;
-    font-weight: bold;
```

Comment:
his will have an effect on Africanlii as well. Are we changing the nav font weight across all the platforms?